packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -547,17 +530,6 @@ function createViewModelHook(elm: HTMLElement, vnode: VCustomElement, renderer:
     }
 
     const { sel, mode, ctor, owner } = vnode;
-
-    setScopeTokenClassIfNecessary(elm, owner, renderer);
-    if (owner.shadowMode === ShadowMode.Synthetic) {
-        const { stylesheetToken } = owner.context;
-        // when running in synthetic shadow mode, we need to set the shadowToken value
-        // into each element from the template, so they can be styled accordingly.
-        if (!isUndefined(stylesheetToken)) {
-            setElementShadowToken(elm, stylesheetToken);
-        }
-    }
```

Comment:
This logic was moved to `mountCustomElement` to be symmetrical with other methods like `mountElement`.